### PR TITLE
ui: handle undefined value when filtering statement execution statistics

### LINKED ISSUE #56252: sql: statements detail page isn't working
When I click on the sessions -> statements detail page for a given session, it won't load:
Screen Shot 2020-11-03 at 12 24 59 PM

Video: https://drive.google.com/file/d/1SeVhS9lvoo79jnQzvTUH2p6GpngiXJGO/view?usp=sharing
Version: V20.2.0-ALPHA.1-4562-GECA8746C4A-DIRTY

Repro:
```SQL
cd go/src/github.com/cockroachdb/cockroach
roachprod create local
roachprod put local cockroach 
roachprod put local ./bin/workload
roachprod start local
http://localhost:26264/#/overview/list
roachprod run local:4 -- "./cockroach workload fixtures import tpcc --warehouses=10 --db=tpcc"
roachprod run local:4 "./workload run tpcc --wait false --tolerate-errors --warehouses=10 --active-warehouses=10 --duration=15m --split --scatter {pgurl:1-3}"
```

### PR DESCRIPTION
**Pull request by asubiotto**
Release note: None (not present in release)

Closes #56252

### CODE REVIEW

File: pkg/ui/src/views/statements/statementDetails.tsx [hunk 1]
```diff
@@ -505,9 +505,12 @@ export class StatementDetails extends React.Component<StatementDetailsProps, Sta
                   format: Bytes,
                 },
               ].filter(function (r) {
-                if (r.name === "Network Bytes Sent" && r.value.mean === 0) {
-                  // Omit if empty.
-                  return false;
+                if (r.name === "Network Bytes Sent" && r.value) {
```
Comment (jordanlewis) [inline pkg/ui/src/views/statements/statementDetails.tsx:508]: Should we instead make sure the stat is always populated on the server side? This feels fine but I don't like how unprincipled the protobufs are - it's like any value could be present or not :/

@dhartunian do you have thoughts here?

Comment (asubiotto) [reply pkg/ui/src/views/statements/statementDetails.tsx:508]: I'm not sure I understand the suggestion. The protobuf field is non-nullable and it is valid for it not to be populated (until always-on tracing is a thing).

Comment (jordanlewis) [reply pkg/ui/src/views/statements/statementDetails.tsx:508]: Hmm, okay. I guess I'm the one who doesn't understand!

**Review by jordanlewis — APPROVED** [review]

File: pkg/ui/src/views/statements/statementDetails.tsx (hunk 1 above)
Comment (asubiotto) [reply pkg/ui/src/views/statements/statementDetails.tsx:508]: Let me answer your questions. There is still one unknown here which is why is `r.value` undefined in the first place (https://github.com/cockroachdb/cockroach/issues/56432). I don't have an answer to that yet.

Comment (asubiotto) [reply pkg/ui/src/views/statements/statementDetails.tsx:508]: @dhartunian has a PR resolving the undefined issue: https://github.com/cockroachdb/cockroach/pull/56445/files. Seems like the PR also resolves the issue this PR tries to fix (specifically accessing an undefined value), so will close this in favor of that one. Let me know if you want to discuss anything else related to this.

**Comment by asubiotto** [comment]
Closing in favor of #56445

**Pull request closed without merging**